src/cleaner/config/cleaner.yaml

```diff
@@ -15,3 +15,5 @@
 # DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+threshold: 94
```

Comment:
A markdown document is needed to describe this service's configuration. Like this: [https://github.com/Microsoft/pai/blob/pai-0.9.y/src/cluster/config/cluster.md](https://github.com/Microsoft/pai/blob/pai-0.9.y/src/cluster/config/cluster.md)

Reply:
Sure, will add it